wire/blockheader.go

```diff
@@ -104,7 +101,7 @@ func NewBlockHeader(prevHash *chainhash.Hash, merkleRootHash *chainhash.Hash,
 	// Limit the timestamp to one second precision since the protocol
 	// doesn't support better.
 	return &BlockHeader{
-		Version:    BlockVersion,
+		Version:    0,
```

Comment:
Fair point.  I thought about that, but I wasn't really sure it's worth breaking the public API over since the version will have to be set regardless (typically after the header is created) by looking at the various threshold states after this goes in.  I'll go ahead and change it though since the only thing that actually even calls this right now are the tests in the peer package.